color-picker-project/src/components/colorpicker/ColorPicker.css

```diff
@@ -0,0 +1,42 @@
+.color-picker-div {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.color-picker-container {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+
+.color-picker-wheel {
+    margin-top: 2rem;
+}
+
+.color-picker-color {
+    margin-bottom: 3rem;
+}
+
+.color-picker-material {
+    width: 200px;
```

Comment:
This needs to be a little wider, see screenshot (136 partially cut off in the "R" field of RGB):

<img width="406" alt="image" src="https://github.com/chingu-voyages/v49-tier2-team-18/assets/106836794/a8b9fbd7-f9dc-47e5-96e9-cdda3d5259cc">